src/common/lib/transport/connectionmanager.ts

```diff
@@ -832,7 +816,7 @@ class ConnectionManager extends EventEmitter {
    * @param transport the transport instance
    * @param connectionId the id of the new active connection
    * @param connectionDetails the details of the new active connection
-   * @param connectionPosition the position at the point activation; either {connectionSerial: <serial>} or {timeSerial: <serial>}
+   * @param connectionPosition the position at the point activation, {connectionSerial: <serial>}
```

Comment:
I think this comment is wrong (and was wrong prior to these changes). The type of `connectionPosition` is `ConnectionManager` for some reason, although it should be a `Message` (not a serial).

Reply:
It's a subtype of both `ConnectionManager` and `Message`, but really the type should have been exactly what the comment says: `{connectionSerial: number} | {timeSerial: string}`

but it only existed to be that union type, now that no longer exists we can just make the param just plain ol' `connectionSerial: number` , no?

Reply:
Oh I just saw this after I already did it a different way, but yeah that makes a lot more sense